### PR TITLE
Feature/us17736 create journal entries from adjustments

### PR DESCRIPTION
**Pull request by jpc11623**

### CODE REVIEW

File: Crossroads.Service.Finance.Test/Exports/ExportServiceTest.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                .Returns(mpAdjustmentsMock);
+
+            _adjustmentRepository.Setup(r => r.UpdateAdjustments(It.IsAny<List<MpDistributionAdjustment>>()));
+
+            _journalEntryRepository.Setup(r => r.CreateOrUpdateMpJournalEntries(It.IsAny<List<MpJournalEntry>>()));
+
+            // Act
+            _fixture.CreateJournalEntries();
+
+            // Assert
```
Comment (vchtelmakh) [inline Crossroads.Service.Finance.Test/Exports/ExportServiceTest.cs]: Not a strong preference, but can we assume that every dev that looks at this is familiar with the arrange / act / assert paradigm and doesn't need the comments?

File: Crossroads.Service.Finance/Controllers/ExportController.cs [hunk 2]
```diff
@@ -1,19 +1,36 @@
...
+        }
 
-    [HttpGet]
-    [Route("adjustjournalentries")]
-    public IActionResult AdjustJournalEntries()
-    {
-      _logger.Info("Running adjust journal entries job...");
-      return Ok();
+        [HttpPost]
+        [Route("journalentries")]
```
Comment (vchtelmakh) [inline Crossroads.Service.Finance/Controllers/ExportController.cs]: `[Route("journalentries/adjust")]` ?

File: Crossroads.Service.Finance/Services/Exports/ExportService.cs [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            _mapper = mapper;
+        }
+
+        public void CreateJournalEntries()
+        {
+            // get adjustments that are not exported
+            var yesterday = DateTime.Now;
+            var startDate = new DateTime(yesterday.Year, yesterday.Month, yesterday.Day);
+            var endDate = yesterday.AddDays(1);
+            var mpDistributionAdjustments = _adjustmentRepository.GetAdjustmentsByDate(startDate, endDate);
```
Comment (vchtelmakh) [inline Crossroads.Service.Finance/Services/Exports/ExportService.cs]: Would put this in a `GetAdjustmentsWhichArentExported` function.

File: Crossroads.Service.Finance/Controllers/ExportController.cs (hunk 2 above)
Comment (jpc11623) [reply Crossroads.Service.Finance/Controllers/ExportController.cs]: Changed

File: Crossroads.Service.Finance.Test/Exports/ExportServiceTest.cs (hunk 1 above)
Comment (jpc11623) [reply Crossroads.Service.Finance.Test/Exports/ExportServiceTest.cs]: I prefer to use that to separate what each part of that is doing.

**Pull request merged**